book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] I suggest you add the logic for creating a new rental here and ensure that the book itself is sent as a parameter to create the new rental achieving the `has-many` association. This will be easier for you as you just need to create a book, person and date then call this method on the created book passing the person and date as the parameter. Link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship. See suggestion below:

```suggestion
  def add_rentals(person, date)
    Rental.new(person, self, date)
  end
```